cardano-cli/src/Cardano/CLI/EraBased/Run/Governance/DRep.hs

```diff
@@ -58,18 +59,23 @@ runGovernanceDRepCmds = \case
     runGovernanceDRepMetadataHashCmd args
       & firstExceptT CmdGovernanceCmdError
 
-runGovernanceDRepKeyGenCmd :: ()
+runGovernanceDRepKeyGenCmd_ :: ()
   => Cmd.GovernanceDRepKeyGenCmdArgs era
   -> ExceptT (FileError ()) IO ()
+runGovernanceDRepKeyGenCmd_ = void . runGovernanceDRepKeyGenCmd
```

Comment:
Please remove this function that is polluting the namespace while providing little value: callers can call void themselves 👍